### PR TITLE
Improve test performance

### PR DESCRIPTION
**Pull request by Sartonon**
Improve test performance by configuring defaultHidden to true.

[More info](https://testing-library.com/docs/dom-testing-library/api-queries#byrole)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
     "build": "react-scripts build",
     "lint": "eslint --ext js,ts,tsx src",
     "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
-    "test:changed": "npm run test -- -o --watchAll=false",
+    "test:changed": "npm test -- -o --watchAll=false",
```
Comment (jorilindell) [inline package.json:72]: Maybe `yarn test -- -o --watchAll=false` to be consistent

**Review by jorilindell — APPROVED** [review]
This is nice one. The difference seems to be significant. e.g Datepicker.test.tsx 20s vs. 80s

**Comment by Sartonon** [comment]
> This is nice one. The difference seems to be significant. e.g Datepicker.test.tsx 20s vs. 80s

Yeah. getByRole does some expensive checking that is not that important.

File: package.json (hunk 1 above)
Comment (Sartonon) [reply package.json]: updated

**Pull request merged**